automation/terraform/testnets/mainnet/genesis_ledger.json

```diff
@@ -0,0 +1,19264 @@
+{
+  "genesis": {
+    "genesis_state_timestamp": "2021-03-06T00:00:00Z"
```

Comment:
This should be set to 4pm PT 3/16

Reply:
this is not the mainnet ledger, its the finalfinal2 ledger with a new genesis timestamp so we can deploy something and run a test before the mainnet ledger is ready.